src/main/resources/static/templates/PasswordResetVulnerability/Common/PasswordReset.html

```
@@ -0,0 +1,51 @@
+    <div id="resultDetails" class="pr-details"></div>
+  </div>
+
+  <div class="pr-card hidden" id="referrerLeakCard">
+    <h5>Third-party Content</h5>
+    <p class="pr-subtitle">Level 7 intentionally loads a third-party image to demonstrate referrer leakage of tokenized URLs.</p>
+    <img
+      id="trackerImage"
+      alt="third-party tracker"
+      width="1"
+      height="1"
+      src=""
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)